docs/source/apis/aws_marketplace.rst

```diff
@@ -0,0 +1,245 @@
+
+
+Part 1: Setting-Up a Virtual Private Cloud (VPC)
+------------------------------------------------
+
+1. **Access VPC Dashboard**:
+
+   - In the AWS Management Console, navigate to the VPC service.
+   - Click on “Your VPCs” in the left navigation pane, then “Create VPC.”
+
+2. **Create VPC**:
+
+   - Select ``VPC only``
+   - Enter a ``Name tag`` for your VPC.
+   - Specify the IPv4 CIDR block (e.g., 10.0.0.0/16).
+
+     - You may leave the IPv6 CIDR block and Tenancy settings as default.
+   - Click “Create VPC” button
+
+.. image:: imgs/AWS/VPC_Step2.png
+  :align: center
+  :alt: create vpc
+
+3. **Create Subnets**:
+
+   - After creating the VPC, click on “Subnets” in the left navigation pane.
+   - Click “Create subnet” and select the VPC you just created from the dropdown menu.
+   - For the first public subnet:
+
+     - Enter a ``Name tag``.
+     - Select an ``Availability Zone``.
+     - Specify the IPv4 CIDR block (e.g., 10.0.1.0/24).
+     - Click ``Add new subnet``.
+   - Repeat the process for the second public subnet with a different Availability Zone and CIDR block (e.g., 10.0.2.0/24).
+
+     - *Note: Each subnet must reside entirely within one Availability Zone and cannot span zones*.
+     - Ref: AWS documentation on `Subnet basics <https://docs.aws.amazon.com/vpc/latest/userguide/configure-subnets.html#subnet-basics>`_.
+   - For the private subnet:
+
+     - Follow the same steps, but choose a different Availability Zone and CIDR block (e.g., 10.0.3.0/24).
+
+   - Click ``Create subnet``.
+
+.. image:: imgs/AWS/VPC_Step3.png
+  :align: center
+  :alt: create subnet
+
+4. **Create Internet Gateway (for Public Subnets)**:
+
+   - Go to “Internet Gateways” in the VPC dashboard.
+   - Click “Create internet gateway,” enter a name, and create.
+   - Attach the newly created internet gateway to your VPC.
+
+.. image:: imgs/AWS/VPC_Step4.png
+  :align: center
+  :alt: create internet gateway
+
+5. **Set Up Route Tables**:
+
+   - Go to “Route tables” in the VPC dashboard.
+   - Enter a ``Name``.
+   - Select the ``VPC`` from Step 2 above.
+   - Click ``Create route table``
+
+.. image:: imgs/AWS/VPC_Step5.png
+  :align: center
+  :alt: create route table
+
+6. **Connecting Route Tables and Internet Gateway**:
+
+   - Go to the VPC set up in Step 2.
+   - Connect the **public subnets** to the **route table** from Step 5.
+
+     - Select the public subnet from Step 3.
+     - Click ``Actions`` button on the top right-hand corner
+     - Select ``Edit route table association``
+     - Repeat the process for the two public subnets.
+
+   - Connect the Route table to Internet Gateway
+
+     - Click the ``route table`` from VPC Details page.
+     - Click ``Edit route``
+     - Click ``Add route``:
+
+   - For the **private subnet**, use the main route table or create a new one without a route to the internet gateway.
```

Comment:
what is the main route table? the one created by default? isn't it already connected to this? ..more explicit directions here